server/routes/bookVisit/visitBookedController.ts

```diff
@@ -5,8 +5,23 @@ export default class VisitBookedController {
 
   public view(): RequestHandler {
     return async (req, res) => {
+      const { bookingConfirmed } = req.session
+
+      const prisonPhoneNumber = bookingConfirmed.prisonCode === 'FHI' ? '01234567890' : '09876543210'
+      const prisonWebsite =
+        bookingConfirmed.prisonCode === 'FHI'
+          ? 'https://www.gov.uk/guidance/foston-hall-prison'
+          : 'https://www.gov.uk/guidance/drake-hall-prison'
```

Comment:
It would be safer to have `prisonPhoneNumber` and `prisonWebsite` empty/undefined if the prison is not Foston/Drake. Otherwise you've got Drake Hall's details going through to the template for **any prison** that is not Foston. It works because you set `supportedPrison` but I'd worry that text could be shuffled in the template and we'd end up showing incorrect information - i.e. it's not really 'fail safe'.